osi_common.proto

```diff
@@ -387,6 +387,125 @@ message LogicalLaneAssignment
     optional double angle_to_lane = 4;
 }
 
+// \brief A bounding box containing a sub-section of a object. 
+//
+// A bounding box representing a sub-section of it's parents overall dimension, 
+// either that of a \c MovingObject or \c StationaryObject .
+//
+// The parent frame of the \c BoundingBox is not relative to the parent object
+// it is associated to, but in the same parent frame as the parent object.
+//
+message BoundingBox
+{
+    // The 3D dimensions of the bounding box.
+    //
+    optional Dimension3d dimension = 1;
+
+    // The 3D position of the bounding box.
+    //
+    // \note The position should be within the same coordinate frame as it's
+    // parent, not relative to coordinate frame of the parent object. The 
+    // position becomes global/absolute if the parent frame is inertial 
+    // (all parent frames up to ground truth).
+    //
+    optional Vector3d position = 2;
+
+    // The 3D orientation of the bounding box.
+    //
+    // \note The orientation should be within the same coordinate frame as it's
+    // parent, not relative to coordinate frame of the parent object. The 
+    // orientation becomes global/absolute if the parent frame is inertial 
+    // (all parent frames up to ground truth).
+    //
+    optional Orientation3d orientation = 3;
+
+    // The type of object contained in the bounding box.
+    //
+    optional Type contained_object_type = 4; 
+
+    // Opaque reference of an associated 3D model of the bounding box.
+    //
+    // \note It is implementation-specific how model_references are resolved to
+    // 3d models. This means the coordinate system, model origin, and model
+    // orientation are also implementation-specific.
+    //
+    optional string model_reference = 5;
+
+    // Definition of different types of object contained within the bounding box
```

Comment:
Is there a reference for this list or is it a result of a working-group brainsorming?
If there are sources, I qould like to see them referenced here.